third_party/2and3/redis/client.pyi

```diff
@@ -317,6 +317,7 @@ class PubSub:
     def get_message(self, ignore_subscribe_messages: bool = ..., timeout: float = ...) -> Optional[Dict[str, Any]]: ...
     def handle_message(self, response, ignore_subscribe_messages: bool = ...) -> Optional[Dict[str, Any]]: ...
     def run_in_thread(self, sleep_time=...): ...
+    def ping(self, message: Optional[_Str] = ...) -> None: ...
```

Comment:
Doesn't this return a string per [the documentation](https://redis.io/commands/ping)?

Reply:
Hi @srittau !

In the current implementation, ping (https://github.com/andymccurdy/redis-py/blob/master/redis/client.py#L3561-L3566) calls execute command which has no return statement (https://github.com/andymccurdy/redis-py/blob/master/redis/client.py#L3397-L3414).

To get the PONG message as in the documentation, you have to call `get_message` after pinging the server